orka_vector_api/helper/gdal_helper.py

```diff
@@ -80,3 +78,42 @@ def get_layers_from_file(app):
     with open(os.path.join(app.instance_path, app.config['ORKA_LAYERS_YML'])) as file:
         layers = yaml.load(file, Loader=yaml.FullLoader)
         return layers
+
+
+def create_gpkg_threaded(app, base_url, job_id, *args):
+    db_props = {
+        'host': app.config['PG_HOST'],
+        'port': app.config['PG_PORT'],
+        'database': app.config['PG_DATABASE'],
+        'user': app.config['PG_USER'],
+        'password': app.config['PG_PASSWORD']
+    }
+    gpkg_path = app.config['ORKA_GPKG_PATH']
+    timeout = app.config['ORKA_THREAD_TIMEOUT']
+
+    if not base_url.endswith('/'):
+        base_url += '/'
+    response_url = f'{base_url}{job_id}'
+    thread = Thread(target=_create_gpkg_threaded, args=(response_url, *args),
+                    kwargs={'timeout': timeout, 'db_props': db_props, 'gpkg_path': gpkg_path})
+    thread.start()
+
+
+def _create_gpkg_threaded(response_url, *args, timeout=None, **kwargs):
+    event = Event()
+    thread = Thread(target=create_geopackage, args=(*args, event), kwargs=kwargs)
+    thread.start()
+
+    killed = False
+    if timeout is not None:
+        thread.join(timeout)
+        if thread.is_alive():
+            killed = True
+        event.set()
+    thread.join()
+    # TODO check if thread threw exception
+    if killed:
+        # TODO check if this works with proxy
```

Comment:
Why is there a proxy involved? Shouldn't this be an internal connection/url?

Reply:
This is a just a leftover that can be removed, as I was trying out different concepts during development.

Generally, flask expects to run on root (`/`), which will not necessarily be the case on different systems. Therefore, we use the `ProxyFix` in `__init__.py`. This is just a reminder to check if the url is properly set, in cases where the app does not run on root.